.github/workflows/release.yml

```
@@ -0,0 +1,99 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+      - '20.[0-9]+.[0-9]+'
+jobs:
+  release:
+    if: "!contains(github.event.head_commit.message, '[skip release]')"
+    name: Release
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          persist-credentials: false
+      - uses: Brightspace/third-party-actions@actions/setup-node
+        with:
+          node-version: '14'
+          cache: 'npm'
+      - name: Install dependencies
+        run: |
+          npm ci
+          npm install frau-publisher --no-save
+      - name: Build
+        run: |
+          npm run build
+          npm run build:lang
+        env:
+          NODE_OPTIONS: '--max-old-space-size=6656'
+      - name: Get Branch Name
+        id: get-branch
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
```

This is a different syntax than I'm used to. Is it written down somewhere?
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

Also, the subshell looks unnecessary.

```suggestion
        run: echo "::set-output name=branch::${GITHUB_REF#refs/heads/}"
```

---

I got it from the accepted answer here:
https://stackoverflow.com/questions/58033366/how-to-get-the-current-branch-within-github-actions